src/Bundle/Resources/config/listeners.yml

```diff
@@ -39,6 +39,14 @@ services:
     tags:
       - { name: 'contao.hook', hook: 'initializeSystem', method: 'onInitializeSystem', priority: 100 }
 
+  netzmacht.contao_toolkit.listeners.register_models:
+    class: Netzmacht\Contao\Toolkit\Bundle\DependencyInjection\Listener\RegisterContaoModelsListener
+    public: true
+    arguments:
+      - []
+    tags:
+      - { name: 'contao.hook', hook: 'initializeSystem', method: 'onInitializeSystem', priority: 101 }
```

Comment:
Any reason why you set priority to `101`? I would use `100` here or do you see any issues?

Reply:
No reason for that ... only a wrong number :-) Will change it!